_posts/2025-04-12-test-markdown.md

```
@@ -0,0 +1,97 @@
+```
+
+# 步骤 3：高级可视化（推荐）
+
+## 3.1 启动Web可视化服务
+
+```shell
+go tool pprof -http=:8080 cpu.prof
+```
+
+```shell
+菜单选项	功能描述
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 5

(MD010, no-hard-tabs)